index.js

```diff
@@ -85,11 +78,11 @@ function magnetURIDecode (uri) {
 
   if (result.dn) result.name = result.dn
   if (result.kt) result.keywords = result.kt
-  if (result['x.pe']) result.peerAddresses = result['x.pe']
 
-  if (typeof result.tr === 'string') result.announce = [result.tr]
-  else if (Array.isArray(result.tr)) result.announce = result.tr
-  else result.announce = []
+  result.announce = []
+  if (typeof result.tr === 'string' || Array.isArray(result.tr)) {
+    result.announce = result.announce.concat(result.tr)
+  }
```

Comment:
This is just code cleanup. I made the style similar to the handling of `urlList` below.